roles/node_images_hypervisor/vars/packages/suse.yml

```diff
@@ -23,30 +23,38 @@
 #
 ---
 packages:
-  # cloud-init often updates and goes missing in repodata, use `<23.2` to grab 23.1 no matter the release.
+  # rationale: Application for faciliatting configuring a hypervisor node.
+  - crucible=0.0.1-1
+  # rationale: SUSE's cloud-init config packages.
   - cloud-init-config-suse<23.2
+  # rationale: Documentation for cloud-init
   - cloud-init-doc<23.2
+  # rationale: Automates server provisioning.
   - cloud-init<23.2
+  # rationale: Free range routing.
   - frr=8.4-lp154.50.1
-  - helm=3.11.2-150000.1.21.1
-  - helm-bash-completion=3.11.2-150000.1.21.1
-  - helm-zsh-completion=3.11.2-150000.1.21.1
-  - gcc-c++=7-3.9.1
-  - libstdc++-devel=7-3.9.1
-  - ruby2.5-devel=2.5.9-150000.4.26.1
-  - ruby3.1-devel=3.1.4-lp154.36.1
+  # rationale: Necessary for Mercury software proof-of-concepts.
   - libguestfs0=1.44.2-150400.3.3.1
-  - libvirt-client=8.0.0-150400.7.3.1
-  - libvirt-devel=8.0.0-150400.7.3.1
-  - libvirt=8.0.0-150400.7.3.1
+  # rationale: Dependency of libvirt.
+  - libvirt-client=8.0.0-150400.7.6.1
+  # rationale: Dependency of libvirt.
+  - libvirt-devel=8.0.0-150400.7.6.1
+  # rationale: Hypervisor virtualization API.
+  - libvirt=8.0.0-150400.7.6.1
+  # rationale: Necessary for Mercury software proof-of-concepts.
   - libxslt-tools=1.1.34-150400.3.3.1
-  - mkisofs=3.02~a09-4.6.1
-  - nfs-kernel-server=2.1.1-150100.10.32.1
+  # rationale: Proof-of-concept item for launching virtual machines.
   - pulumi-3.73.0-2
```

Comment:
Do we need pulumi and terraform in the hypervisor anymore? mkisofs?

Reply:
I couldn't find where `mkisofs` was being used, it's not even used in the `create-iso-artifacts.sh` script.

I'll remove `pulumi` and `terraform`.